pkgs/development/compilers/gambit/build.nix

```diff
@@ -1,41 +1,47 @@
-{ stdenv, git, openssl, autoconf, pkgs, makeStaticLibraries, version, gcc, src, coreutils }:
+{ gccStdenv, lib, git, openssl, autoconf, pkgs, makeStaticLibraries, gcc, coreutils, gnused, gnugrep,
+  src, version, git-version,
+  gambit-support, optimizationSetting ? "-O1", gambit-params ? pkgs.gambit-support.stable-params }:
```

Comment:
Is `gambit-params` parameter just ignored in this file?

Reply:
I had commented out the one use, pending debugging of other issues. It's back.

Reply:
Right now Github shows me that it is still commented out. OK, I see it will be uncommented in the final version.